test/trustedresources.go

```diff
@@ -230,3 +363,30 @@ func readPasswordFn(confirm bool) func() ([]byte, error) {
 		return nil, fmt.Errorf("fail to get password")
 	}
 }
+
+// PassFunc is the function to be called to retrieve the signer password. If
+// nil, then it assumes that no password is provided.
+type PassFunc func(bool) ([]byte, error)
+
+func pass(s string) PassFunc {
```

Comment:
nit: can you rename this to something like `generatePassword`? from the name I thought it was doing something around making the test pass

Reply:
sure! :D